db_stress_tool/db_stress_test_base.h

```diff
@@ -281,6 +281,7 @@ class StressTest {
   }
 
   bool IsRetryableInjectedError(const Status& s) const {
+    assert(!s.ok());
```

Comment:
I guess it's not compatible with this assertion. Was this assertion useful in getting the logic correct?

Reply:
A more technically correct name with this assertion would be `IsErrorInjectedAndRetryable`

Reply:
Will fix this thanks!

Reply:
Fixed